lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -1085,7 +1086,7 @@ sub save_user {
     try { $user->create($request->{password}); }
     catch {
         if ($_ =~ /duplicate user/i){
-           $request->{dbh}->rollback;
+           $request->{dbh}->rollback; # Rolling back because of the error
```

Comment:
This too is an exact description of what the statement does. Is there anything you want to add here that's not visible on the programmer's screen when (s)he looks at the code? Then that would be a helpful comment, as per the quote from Chris's description above.